base/src/main/java/uk/ac/ebi/atlas/search/suggester/SuggesterService.java

```diff
@@ -1,18 +1,24 @@
 package uk.ac.ebi.atlas.search.suggester;
 
 import com.google.common.collect.ImmutableMap;
+import org.apache.solr.client.solrj.response.Suggestion;
 import org.springframework.stereotype.Component;
 import uk.ac.ebi.atlas.species.Species;
 import uk.ac.ebi.atlas.species.SpeciesFactory;
 
 import java.util.Arrays;
 import java.util.Map;
+import java.util.function.Function;
 import java.util.stream.Stream;
 
 @Component
 public class SuggesterService {
     // Remember that suggestions are distinct()’ed, so this value is an upper bound
-    private static final int DEFAULT_MAX_NUMBER_OF_SUGGESTIONS = 20;
+    public static final int DEFAULT_MAX_NUMBER_OF_SUGGESTIONS = 20;
+
+    private static final Function<Suggestion, Map<String, String>>
```

Comment:
Hmm adding `Suggestion` here couples this service to Solr, when it fact it should be the Dao who is concerned about where the suggestions are coming from.

Reply:
I agree in principle. The reason why I didn’t was that, after looking at [Solr’s `Suggestion` class](https://github.com/apache/lucene-solr/blob/1d85cd783863f75cea133fb9c452302214165a4d/solr/solrj/src/java/org/apache/solr/client/solrj/response/Suggestion.java), it seemed a bit superfluous to create pretty much the same thing under our own namespace. It’s just a data class with no behaviour.